scss/_reboot.scss

```diff
@@ -96,13 +96,13 @@ body {
   @include focus-visible(); // 1
 }
 
-.js-focus-visible :focus:not([data-focus-visible-added]):not(.focus-ring),
-.js-focus-visible .focus:not([data-focus-visible-added]):not(.focus-ring) { // 2
+.js-focus-visible :focus:not([data-focus-visible-added]):not(.focus-ring):not(.form-select):valid,
+.js-focus-visible .focus:not([data-focus-visible-added]):not(.focus-ring):not(.form-select):valid { // 2
   outline: 0 !important;
   box-shadow: none;
 }
 
-:focus:not(:focus-visible):not(.focus-ring) { // 3
+:focus:not(:focus-visible):not(.focus-ring):not(.form-select):valid { // 3
```

Comment:
Still not sure about this part 🤔 

We should not have to changed this file but Chome is not doing what he is supposed to do and Firefox is doing what he is not supposed to do... 

Still need more investigations...